lib/go/replication/replication.pb.go

```diff
@@ -1,20 +1,16 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
Explain in the commit how these files are generated. You added a make target, so the command in the commit should be `make ...`.